webapi/src/main/scala/org/knora/webapi/routing/HandlerMapperF.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright © 2021 - 2023 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.routing
+
+import sttp.tapir.Endpoint
+import sttp.tapir.server.ServerEndpoint.Full
+import zio.Task
+import zio.ZIO
+import zio.ZLayer
+
+import scala.concurrent.Future
+
+import dsp.errors.RequestRejectedException
+
+case class EndpointAndZioHandler[INPUT, OUTPUT](
+  endpoint: Endpoint[Unit, INPUT, RequestRejectedException, OUTPUT, Any],
+  handler: INPUT => Task[OUTPUT]
+)
+
+final case class HandlerMapperF()(implicit val r: zio.Runtime[Any]) {
+
+  def mapEndpointAndHandler[INPUT, OUTPUT](
+    it: EndpointAndZioHandler[INPUT, OUTPUT]
+  ): Full[Unit, Unit, INPUT, RequestRejectedException, OUTPUT, Any, Future] =
+    it.endpoint.serverLogic[Future](handlerFromZio(it.handler))
```

Comment:
nicer name for `it`?

Reply:
done => 95d48c809544908b61bbe0e1d4749b6f80f3ce3c